tests/test_check_stac_metadata.py

```diff
@@ -106,6 +105,7 @@ def should_save_multiple_directories_validation_results(
     version_id = any_dataset_version_id()
     base_url = f"s3://{staging_bucket_name}/"
     invalid_key = f"test/{any_safe_filename()}"
+    valid_key = any_safe_filename()
```

Comment:
We should probably make *both* asset keys invalid, and then assert that we save *two* multiple directories errors.